EKF/control.cpp

```diff
@@ -341,14 +343,13 @@ void Ekf::controlFusionModes()
 		_control_status.flags.rng_hgt = true;
 	}
 
-	// Placeholder for control of wind velocity states estimation
-	// TODO add methods for true airspeed and/or sidelsip fusion or some type of drag force measurement
-	if (false) {
+	// if the airspeed measurements have timed out for 10 seconds we declare the wind estimate to be invalid
```

Comment:
@priseborough @CarlOlsson At the moment I have a hardcoded factor of 10 seconds for the airspeed sensor to timout until we declare wind states to be invalid.